docs/experiments/multiple-focus-metavariables.md

```diff
@@ -0,0 +1,39 @@
+---
+slug: multiple-focus-metavariables
+append_help_link: true
+description: "With this rule, Semgrep matches all pieces of code captured by the focus metavariables."
+---
+
+# Using multiple focus metavariables
+
+Use multiple focus metavariables in one Semgrep rule by specifying each under a new `focus-metavariable` item in a YAML list. With this rule, Semgrep matches all pieces of code captured by the focus metavariables.
```

Comment:
Use multiple focus metavariables in one ~~Semgrep rule~~ by specifying each under a new `focus-metavariable` item in a YAML list

Reply:
```suggestion
With this syntax, Semgrep matches all pieces of code captured by the focus metavariables. Specify each metavariable that you want to focus by including it in a YAML list.
```